Dev-10/ArrayMaker.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+
+namespace Dev_10
+{
+    // Class works with an array of double values.
+    class ArrayMaker
+    {
+        const int MINVALUE = 2;
+        const int MAXVALUE = 10;
+        const int MINAMOUNTOFELEMENTS = 1;
+        const double EPS = 1e-3;
+        const int NUMBAFTERCOMMA = 3;
+        const string TEXTFORZEROREPEATINGS = "There are no repeatings in the N arrays.";
+        public double[] RepeatingElementsArray { get; set; }
+        public double[][] DoubleValuesArray { get; set; }
+        public int AmountOfArrays { get; set; }
+        public int AmountOfDoubleValues { get; set; }
+
+        //Creates array with repeating elements.
+        //Return true if there are repeating elements, else - false.
+        public bool Create()
```

Comment:
Bad name, it is not create a new array, it checks if elements equals, not only create